<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login</class>
 <widget class="QWidget" name="login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>969</width>
    <height>834</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>40</y>
     <width>300</width>
     <height>461</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>1</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
    background-image: url(:/images/myimage.png);
    background-repeat: no-repeat;
    background-position: center;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="LOGO.qrc">:/images/coucoueats_logo.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="id_le"/>
      </item>
      <item>
       <widget class="QLineEdit" name="pass_le"/>
      </item>
      <item>
       <widget class="QPushButton" name="login_btn">
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #F5F5F5;
    border: 2px solid #D3D3D3;
    border-radius: 30px;
    font-size: 16px;
    color: black;
}

QPushButton:hover {
    background-color: #E0E0E0;
    border: 2px solid #B0B0B0;
}

QPushButton:pressed {
    background-color: #C0C0C0;
    border: 2px solid #A0A0A0;
}</string>
        </property>
        <property name="text">
         <string>로그인</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="join_btn">
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #F5F5F5;
    border: 2px solid #D3D3D3;
    border-radius: 30px;
    font-size: 16px;
    color: black;
}

QPushButton:hover {
    background-color: #E0E0E0;
    border: 2px solid #B0B0B0;
}

QPushButton:pressed {
    background-color: #C0C0C0;
    border: 2px solid #A0A0A0;
}</string>
        </property>
        <property name="text">
         <string>판매자 회원가입</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>사업주 로그인</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="find_id_btn">
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #F5F5F5;
    border: 2px solid #D3D3D3;
    border-radius: 30px;
    font-size: 16px;
    color: black;
}

QPushButton:hover {
    background-color: #E0E0E0;
    border: 2px solid #B0B0B0;
}

QPushButton:pressed {
    background-color: #C0C0C0;
    border: 2px solid #A0A0A0;
}</string>
        </property>
        <property name="text">
         <string>아이디찾기</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="find_pass_btn">
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #F5F5F5;
    border: 2px solid #D3D3D3;
    border-radius: 30px;
    font-size: 16px;
    color: black;
}

QPushButton:hover {
    background-color: #E0E0E0;
    border: 2px solid #B0B0B0;
}

QPushButton:pressed {
    background-color: #C0C0C0;
    border: 2px solid #A0A0A0;
}</string>
        </property>
        <property name="text">
         <string>비밀번호 찾기</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="LOGO.qrc"/>
 </resources>
 <connections/>
</ui>
